### DISCUSSION TITLE
Having Trouble implementing mastadon apis.

### DISCUSSION THREAD

**Discussion by FXE8NR6** [mastodon/mastodon#28150] [General]
I have a React application with a Node.js backend. I've implemented authorization directly in the React application, which brings a code on a callback URL. Now, when I send that code to the backend, it fetches the token and then attempts to get followers. However, I encounter the following error whenever I try to fetch. Can anybody help with the code?

<img width="1100" alt="Screenshot 2023-11-30 at 11 27 07 PM" src="https://github.com/mastodon/mastodon/assets/106158699/7a11598a-1815-44d2-89d4-e5d9080b4c7f">

**Comment by andypiper** [comment]
Are you using a specific [JavaScript library](https://docs.joinmastodon.org/client/libraries/) for accessing the Mastodon API? Can you provide a code snippet that demonstrates the issue you're describing?
